### ISSUE TITLE
Redesign of categories icons

### ISSUE THREAD

**Issue by wa4557** [numixproject/numix-icon-theme#1262]
I think at some point we might have to do a redesign of the `categories` icons. The current ones are nice and all, but do not really fit with the rest of the theme. A bit more "material design" would be a cool thing...

**Comment by Foggalong** [comment]
Yeah they definitely do. The theme has been updated bit by bit over the years so some sections are massively more out of date than others (`categories/` being most out of date, `devices/` being most recently updated)
